rerun-failed-run/README.md

```diff
@@ -46,4 +48,31 @@ jobs:
           vault-addr: ${{ secrets.VAULT_ADDR }}
           vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
           vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+          notify-back-on-error: "true"
+```
+
+
+### Integration Example on the Calling Side
```

Comment:
sorry, it's really confusing to me how to understand this section (what is calling side here)

Can you give a full-blown example somewhere of how this should be used here as a comment? (for my better understanding)